neard/src/cli.rs

```diff
@@ -90,10 +94,14 @@ pub(super) enum NeardSubCommand {
     /// Runs NEAR node
     #[clap(name = "run")]
     Run(RunCmd),
-    /// Sets up testnet configuration with all necessary files (validator key, node key, genesis
-    /// and config)
+    /// Sets up local configuration with all necessary files (validator key, node key, genesis and
+    /// config)
+    #[clap(name = "localnet")]
+    Localnet(LocalnetCmd),
+    /// DEPRECATED: this command has been renamed to 'localnet' and will be removed in a future
+    /// release.
```

Comment:
```suggestion
    /// release.
    // TODO(#4372): Deprecated since 1.24.  Delete it in couple of
    // releases in 2022.
```

Reply:
Done, also added this note to the relevant python test files.

Reply:
I don’t think it’s really needed in Python tests since removing `testnet` subcommand here will force change in Python code as well but either way is fine.

Reply:
I don’t think it’s really needed in Python tests since removing `testnet` subcommand here will force change in Python code as well but either way is fine.